src/twisted/python/_release.py

```diff
@@ -39,14 +40,50 @@
 
 
 def runCommand(args, **kwargs):
-    """Execute a vector of arguments.
+    """
+    Execute a vector of arguments.
 
     This is a wrapper around L{subprocess.check_output}, so it takes
     the same arguments as L{subprocess.Popen} with one difference: all
     arguments after the vector must be keyword arguments.
     """
     kwargs['stderr'] = STDOUT
-    return check_output(args, **kwargs)
+    output = check_output(args, **kwargs)
+    if not isinstance(output, str):
```

Comment:
Well, yes, but what I mean is:

1. this should be documented in its docstring as an `@rtype`
2. its type, specifically, should be documented.  is it supposed to be `L{bytes}` or `L{unicode}`?